.kokoro/run/websockets.cfg

```diff
@@ -0,0 +1,20 @@
+# Format: //devtools/kokoro/config/proto/build.proto
+
+# Set the folder in which the tests are run
+env_vars: {
+    key: "PROJECT"
+    value: "run/websockets"
+}
+
+# Tell the trampoline which build file to use.
+env_vars: {
+    key: "TRAMPOLINE_BUILD_FILE"
+    value: "github/nodejs-docs-samples/.kokoro/build-with-run.sh"
+}
+
+# Configure the docker image for kokoro-trampoline.
+# Override common.cfg to include puppeteer
+env_vars: {
+    key: "TRAMPOLINE_IMAGE"
+    value: "gcr.io/cloud-devrel-kokoro-resources/node:10-puppeteer"
```

Comment:
Could we use a newer version of node.js here?  Current latest LTS is 14, and 10 is technically EOL.

Reply:
tag 12-puppeteer doesn't have gcloud installed and there is no tag for v14. I will add an issue to that repo.